code/modules/clothing/masks/gasmask.dm

```diff
@@ -123,6 +124,7 @@
 	name = "polychromic clown wig and mask"
 	desc = "A true prankster's facial attire. A clown is incomplete without his wig and mask."
 	clothing_flags = ALLOWINTERNALS
+	flags_inv = HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
```

Comment:
Inherits from line 6, this isn't really necessary

Reply:
i forgor to include the actual hides ears

this is what you call a skull moment

Reply:
you still didn't actually change the line commented on here @zeroisthebiggay

Reply:
@zeroisthebiggay see above